extensions/adobe/experience/offer-management/schema-projection.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/offer-management/schema-projection",
```

Comment:
@lrosenthol @trieloff @jbeckert We need a decision on naming / placement for this schema: this is exactly what it is named - a schema projection consisting from the id of an underlying XDM schema and the subset of blessed properties. The instances are run-time objects, so we do not expect to produce a new XDM schema and register it in Registry / Catalog. The admin workflow allows customers to modify these objects.

Reply:
OK, that helps me a lot!   Thanks @dmitry-shkolnik .  

Then using the term schema seems quite appropriate.  Not thrilled with projection, but if you tell me that's a term of art in your industry, then so be it.

Reply:
However, I don't understand the properties bit.  Any properties that could be used here need to be defined here in the schema or another schema. You can't just use an array of strings, as that has no semantic meaning.

Reply:
... but without the filter on the rows. These are only column selections if you want to use the table metaphor. I think `selectedProperties` would be more succinct than `publishedProperties`.

Reply:
I don't like "selected", because then user will select something from already selected. Maybe "whitelisted" or "selectable"? But "published" describes the workflow quite well, in my opinion.

Reply:
how about visibleProperties or allowedProperties, since that's what you are implying?

Reply:
`selectableProperties` or `allowedProperties` both works for me. The aspect that we want to convey here is that those are the properties that are available to be used in decisions but these are also the properties for which values must be provided to the decision engine at runtime; even if those values are null, what matters is that caller makes every attempt to provide those properties.

Reply:
now it's `selectableProperties`

Reply:
now it's `model-usage.schema.json`